ts/features/wallet/onboarding/privative/saga/orchestration/addPrivativeToWallet.ts

```diff
@@ -44,9 +47,11 @@ function* privativeWorkUnit() {
  * to the wallet after the insertion.
  */
 export function* addPrivativeToWalletGeneric() {
```

Comment:
Is this function used?

Reply:
removed in a7c4906789c30fba5e17bb17e6bf0db2adefa669